docs/ab-testing.md

```diff
@@ -0,0 +1,74 @@
+# AB Testing Guide
```

Comment:
👏  great docs !

Reply:
+1! And I am very excited to see this being implemented in another platform 🎊